mkdocs.yml

```diff
@@ -0,0 +1,67 @@
+site_name: Remarx Documentation
+site_description: Code for identifying Marx quotes associated with the CDH project Citing Marx
+site_author: Center for Digital Humanities at Princeton
+site_url: https://github.com/Princeton-CDH/remarx
```

Comment:
When I run `mkdocs serve` the url is http://127.0.0.1:8000/Princeton-CDH/remarx/ - wondering if it is because of this configuration or something else. 

When we start building the docs and pulbishing to github pages, the url will be https://princeton-cdh.github.io/remarx/